lib/ansible/modules/packaging/language/maven_artifact.py

```diff
@@ -112,40 +125,46 @@
     group_id: junit
     artifact_id: junit
     dest: /tmp/junit-latest.jar
-
 # Download JUnit 4.11 from Maven Central
 - maven_artifact:
     group_id: junit
     artifact_id: junit
     version: 4.11
     dest: /tmp/junit-4.11.jar
-
 # Download an artifact from a private repository requiring authentication
 - maven_artifact:
     group_id: com.company
     artifact_id: library-name
-    repository_url: 'https://repo.company.com/maven'
+    repository_url: https://repo.company.com/maven
     username: user
     password: pass
     dest: /tmp/library-name-latest.jar
-
 # Download a WAR File to the Tomcat webapps directory to be deployed
 - maven_artifact:
     group_id: com.company
     artifact_id: web-app
     extension: war
-    repository_url: 'https://repo.company.com/maven'
+    repository_url: https://repo.company.com/maven
+    dest: /var/lib/tomcat7/webapps/web-app.war
+# Copy a WAR file from the local maven repository to the Tomcat webapps directory to be deployed
+- maven_artifact:
+    group_id: com.company
+    artifact_id: web-app
+    extension: war
+    repository_url: file://home/jenkins/.m2/repository
     dest: /var/lib/tomcat7/webapps/web-app.war
 '''
 
-from lxml import etree
+import lxml.etree as etree
 import os
 import hashlib
 import sys
 import posixpath
-import urlparse
+import shutil
+import distutils.version
 from ansible.module_utils.basic import *
 from ansible.module_utils.urls import *
+import urlparse # Override urlparse from module_utils
```

Comment:
Please import what you need rather than `*` 

https://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#python-imports